### DISCUSSION TITLE
Open cart with javascript function

### DISCUSSION THREAD

**Discussion by ChristopherVigh** [EvgeniyMukhamedjanov/liquid-ajax-cart#50] [Ideas]
Hello,

I am working on a feature where i would like to open the cart after a javascript function have been complete, but i can't find a way to get the cart to open with only a javascript function. Would that be something you would implement?

**Comment by EvgeniyMukhamedjanov** [comment]
Hello.
Usually cart gets open when a special class exists on the body tag. So you just need to add this class via Javascript.

**Comment by ChristopherVigh** [comment]
Perfect, i will try that :)


Den ons. 19. okt. 2022 kl. 11.57 skrev EvgeniyMukhamedjanov <
***@***.***>:
